corehq/apps/export/export.py

```diff
@@ -378,7 +378,7 @@ def write_export_instance(writer, export_instance, documents, progress_tracker=N
             DownloadBase.set_progress(progress_tracker, row_number + 1, documents.count)
 
     end = _time_in_milliseconds()
-    tags = ['format:{}'.format(writer.format)]
+    tags = ['format:{}'.format(writer.format), 'export_id:{}'.format(export_instance.get_id)]
```

Comment:
This isn't a good idea since it will increase our custom metric count by a large factor (https://docs.datadoghq.com/developers/metrics/custom_metrics/). If we want to track the duration per export then we should save it to the export doc or somewhere else that we can query.

Reply:
@snopoke i was thinking of using this just like we do for [form submissions](https://app.datadoghq.com/dash/256236/change-feeds-pillows?live=true&page=0&is_auto=false&from_ts=1541418647571&to_ts=1542023447571&tile_size=m&fullscreen=211359693) and look for top 5 domains? That wont be feasible with this?

I like the idea of putting it on the export doc and then use it for anticipating their run time on the next run so might go for that as well.

Reply:
Domain's has a much lower cardinality than export IDs (and even there it's not ideal)

Reply:
Okay, removed